package.json

```diff
@@ -102,5 +101,6 @@
     "url": "https://github.com/RetroAchievements/api-js/issues"
   },
   "homepage": "https://github.com/RetroAchievements/api-js#readme",
-  "author": "RAWeb Team"
+  "author": "RAWeb Team",
+  "packageManager": "pnpm@9.1.1+sha256.9551e803dcb7a1839fdf5416153a844060c7bce013218ce823410532504ac10b"
```

Comment:
added `packageManager`
why?
The same comment from the workflow file, pnpm sometimes breaks between patch/minor updates.